### ISSUE TITLE
Few things - may already be covered

### ISSUE THREAD

**Issue by pqmertz** [roygreenfeld/django-crash-course#18]
3. Type ls on Mac ordir on  (or dir)

26.3 Move It Into the Project
mv cheeses everycheese/
gives a permission denied error in vsCode
I copied and deleted...  Really not sure about this strategy though of moving all apps in this dir.

**Comment by pydanny** [comment]
> Type ls on Mac ordir on (or dir)

Nice catch!

> Move It Into the Project

Weird, this isn't a problem for us. Maybe we should supply the command-line actions instead?

> Strategy on moving into projects

The reason behind our strategy of moving apps into one folder is this: On significant projects the directory with apps can get large. If apps are then also mixed in with config, deployment, templates, docs, and more on the same level, it can be hard to navigate. That said, it's a stylistic decision on our part and a lot of people like to have everything in one directory. 

Perhaps we should describe the reason we provide above? Any thougths @pqmertz

**Comment by pqmertz** [comment]
Well...    Given that is where you are putting the users in the projectTitledDir subdirectory it makes sense.   I personally prefer an accounts app for the users, and adding a profile as well.   (Do have a question why you are not going that route vs adding everything to the actual user.   Especially for simple things like a Bio?)

Also, I typically leave the settings under the project titled directory, where you have an upper level directory.    As you say....    a style...    Yours is growing on me quickly (especially the config upper level directory) As someone new to Django, I have read and read, bought Udemy courses etc. etc.    The problem with all this is...   I really wanted to go all in with CBV and/or gCBV.    Most started with functions and you end up with a project that is all mixed together and not a real good future reference.   AND you have to deal with everyone's different styles.

I LOVE that you are only doing CBV and the newest versions.   Also, that you seem to explain some of the "obvious" things that I eventually figured out on my own.   BUT sure would have been nice to just been told.   OH and saying how to do templete tags right away was like...   Thank God someone stated that.   As someone else stated.   An idea of the future "chapters" would be nice.   AND do we have to wait for the end to buy the "extensions"?

In conclusion...     Yes, all a matter of style...    A 20.1 area explaining your style and why would be beneficial.

**Comment by pqmertz** [comment]
OH... and permission denied may be related to me running a WSL (so I'm actually running Linux) vs an Apple or straight Microsoft setup.

**Comment by pqmertz** [comment]
One other thing...   Django 3.0.2 cost me a day of head scratching...   I had an abstract class with children below in the cheese type directory and weird stuff just kept on happening.   Undefined this, no app names etc. etc.

Upgrading to 3.0.3 seemed to address these strange happenings.    Just a heads up for others, and I think your requirements call out 3.0.2

**Comment by pydanny** [comment]
Thank you so much for all your feedback!

We don't specifically name a version of Django to use. 

In any case, I'm closing this issue so we can more easily track items and people can avoid filing duplicates. 😄
